packages/ui/package.json

```diff
@@ -81,6 +81,7 @@
     "test": "yarn update-storybook-imports && yarn lint:js && react-scripts test --env=jsdom",
     "test:coverage": "npm test -- --coverage",
     "test:unit:watch": "react-scripts test --env=jsdom --watch",
+    "test:unit:fast": "REACT_APP_TEST_UI=false react-scripts test --env=jsdom --watch",
```

Comment:
REACT_APP_UI_TESTS for consistency? others are called API_TESTS, DB_TESTS

Reply:
It seems like some component tests are run despite using this comman:
![screenshot 2018-03-05 14 41 38](https://user-images.githubusercontent.com/12738484/36978013-69ce11ba-2083-11e8-8304-dc9701093ffa.png)